faces/3.0/_index.md

```diff
@@ -37,20 +37,25 @@ Please reference that ballot for the official results.
 
 ## Release Review
 
-The Specification Committee Ballot concluded successfully on 2020-mm-dd with the following results.
+The Specification Committee Ballot concluded successfully on 2020-10-21 with the following results.
 
 | Representative                                      | Representative for: | Vote |
 |-----------------------------------------------------|---------------------|------|
-| Kenji Kazumura                                      | Fujitsu             |      |
-| Dan Bandera, Kevin Sutter                           | IBM                 |      |
-| Ed Bratt, Dmitry Kornilov                           | Oracle              |      |
-| Andrew Pielage, Matt Gill                           | Payara              |      |
-| Scott Stark, Mark Little                            | Red Hat             |      |
-| David Blevins, Jean-Louis Monteiro, Cesar Hernandez | Tomitribe           |      |
-| Ivar Grimstad                                       | EE4J PMC            |      |
-| Marcelo Ancelmo, Martijn Verburg                    | Participant Members |      |
-| Werner Keil                                         | Committer Members   |      |
-| Scott (Congquan) Wang                               | Enterprise Members  |      |
-|                                                     | Total               |      |
+| Kenji Kazumura                                      | Fujitsu             |   +1   |
+| Dan Bandera, Kevin Sutter                           | IBM                 |   +1   |
+| Ed Bratt, Dmitry Kornilov                           | Oracle              |   +1   |
+| Andrew Pielage, Matt Gill                           | Payara              |   +1   |
+| Scott Stark, Mark Little                            | Red Hat             |   +1   |
+| David Blevins, Jean-Louis Monteiro, Cesar Hernandez | Tomitribe           |   +1   |
```

Comment:
Nit... Cesar is not a voting member from Tomitribe, his name should be removed from this list.